project/ProjectPlugin.scala

```diff
@@ -22,6 +22,7 @@ object ProjectPlugin extends AutoPlugin {
       val fs2ReactiveStreams: String = "0.5.0"
       val grpc: String               = "1.9.0"
       val monix: String              = "3.0.0-M3"
+      val netty: String              = "2.0.7.Final"
```

Comment:
Should we use something different than "netty", it is a tiny bit confusing that is not the netty version, but the version of netty-tcnative-boringssl-static.